CHANGELOG.md

```diff
@@ -26,6 +28,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Dimming colors which use the indexed `CSI 38 : 5 : Ps m` notation
 - Slow rendering performance with a lot of cells with underline/strikeout attributes
 - Performance of scrolling regions with offset from the bottom
+- Extra mouse buttons are no longer ignored on Wayland
+- Numpad arrow keys are now properly recognized on Wayland
+- Compilation when targetting aarch64-apple-darwin
```

Comment:
@chrisduerr curious when 0.7.0 lands, will arm64 be supported / prebuilt on Macs? The way that the changelog is written makes it sounds like it is really just "fixing" compilation, while no official support will land yet. Is that correct? Thank you.

Reply:
ARM should be supported, but we will not provide prebuilt binaries. Fixing compilation is all there is to do for Alacritty to support a platform.